### DISCUSSION TITLE
Flit doesn't install rich by default

### DISCUSSION THREAD

**Discussion by svlandeg** [fastapi/typer#792] [Questions]
### First Check

- [X] I added a very descriptive title here.
- [X] I used the GitHub search to find a similar question and didn't find it.
- [X] I searched the Typer documentation, with the integrated search.
- [X] I already searched in Google "How to X in Typer" and didn't find any information.
- [X] I already read and followed all the tutorial in the docs and didn't find an answer.
- [X] I already checked if it is not related to Typer but to [Click](https://github.com/pallets/click).

### Commit to Help

- [X] I commit to help with one of those options 👆

### Example Code

```python
flit install --deps develop --pth-file
pip list
```

### Description

I followed the development installation guide: https://typer.tiangolo.com/contributing/

and on my Windows system, ran
```
flit install --deps develop --pth-file
```

Which worked fine. However, this doesn't install `rich` or `shellingham` by default - is that expected behaviour? Because these dependencies do get included when you just `pip install typer`.

### Operating System

Windows

### Typer Version

0.12.1

### Python Version

3.10.5

**Comment by kinuax** [comment]
I also encountered the same situation. Test dependencies need to be installed separately as well. Anyway, Flit was replaced with PDM. Please, see https://github.com/tiangolo/typer/discussions/831.
